Resources/views/frontend/_public/src/js/jquery.swag-paypal-unified.in-context-checkout.js

```diff
@@ -69,28 +70,36 @@
         /**
          * Creates the PayPal in-context button with the loaded PayPal javascript
          */
-        createButton: function() {
+        createButton: function () {
             var me = this,
                 $head = $('head');
 
+            this.payPalObjectInterval = setInterval(this.payPalObjectCheck.bind(this), this.opts.interval);
             if (!$head.hasClass(this.opts.paypalScriptLoadedSelector)) {
                 $.ajax({
                     url: this.renderSdkUrl(),
                     dataType: 'script',
                     cache: true,
-                    success: function() {
+                    success: function () {
                         $head.addClass(me.opts.paypalScriptLoadedSelector);
                         me.paypal = window.paypal;
                         me.renderButton();
                     }
                 });
-            } else {
-                this.paypal = window.paypal;
-                this.renderButton();
             }
         },
 
-        renderSdkUrl: function() {
+        payPalObjectCheck: function () {
```

Comment:
is it possible to abstract this method? as it is used in every plugin

Reply:
like discussed